mets_adapter/mets_adapter/src/main/scala/uk/ac/wellcome/mets_adapter/models/Bag.scala

```diff
@@ -10,17 +10,19 @@ case class Bag(info: BagInfo,
                version: String,
                createdDate: Instant) {
 
-  def metsLocation: Either[Exception, MetsLocation] =
+  def metsLocation: Either[Exception, MetsSourceData] =
     file
       .flatMap { file =>
         parsedVersion.map { version =>
-          MetsLocation(
-            location.bucket,
-            location.path,
-            version,
-            file,
-            createdDate,
-            manifestations)
+          MetsSourceData(
+            bucket = location.bucket,
+            path = location.path,
+            version = version,
+            file = file,
+            createdDate = createdDate,
+            deleted = manifest.files.forall(f => f.path == file),
```

Comment:
Might be worth a comment here explaining that "only the METS file" => deleted, with a link to the ticket/discussion.